custom_components/tahoma/sensor.py

```diff
@@ -45,6 +45,10 @@ def __init__(self, tahoma_device, controller):
 
         super().__init__(tahoma_device, controller)
 
+    async def async_added_to_hass(self):
```

Comment:
Is this something that should be added to the TahomaDevice class maybe? It looks like this is also an issue for the light.py and cover.py integrations.

Reply:
Haven't tested it on other devices than sensors but it would make sense.